src/components/hex-map/hex-map.js

```diff
@@ -87,10 +93,14 @@ export default class HexMap extends Component {
   }
 
   componentDidUpdate() {
+    const { showEntityCount, showSystemName, showSystemNumber } = this.props;
```

Comment:
The destructured props are going into hex canvas already so there's no need to explicitly pass them here.

Reply:
I know, but `eslint` marked it as an error because I didn't use those properties, so I added this to use them somewhere, because I didn't want to change the `eslint` rules.

Reply:
Was it unhappy about the proptype definitions that weren't used?

Reply:
yes

Reply:
aslo i had to add:
"linebreak-style": ["error", "windows"]
to the rules (locally), because on windows it is "standard" to check out windows style and push unix style so my local files are windows style and the default rule is unix.

Reply:
You don't need to add them to the prop types if you are just passing it through to the hex canvas function.

Reply:
Fixed in Irame/sectors-without-number@672bee1fe785222cdea0ca94a1e1c35f57f8df00